[KB-176] Supervisor feedback part 3 (#138)
chhoumann/thesis-chemcam: report_thesis/src/sections/baseline_replica.tex

Reviewer: We could refer back to the matrices in the problem definition in the added text. What do you think?

Author: Done

\section{Baseline \& Replica}\label{sec:baseline_replica}
For analyzing Martian geological samples, the \gls{chemcam} team currently uses the \gls{moc} model~\cite{cleggRecalibrationMarsScience2017}.
This model integrates \gls{pls} and \gls{ica} to predict the composition of major oxides.

As shown in figure \ref{fig:moc_pipeline}, the input to the \gls{moc} model is \gls{ccs} data.
This spectral data is collected on Earth in a laboratory setting simulating the Martian environment.
The instrument used to collect this data is a \gls{libs} instrument replicating the \gls{chemcam} instrument on the Curiosity rover.
Both the \gls{chemcam} and laboratory instrument consist of three spectrometers, each producing 2048 channels.
These spectrometers are used to capture the \gls{uv}, \gls{vio}, and \gls{vnir} regions of the spectrum.

> For each sample, five \gls{ccs} datasets are collected by shooting 50 times at the sample from different locations.

Reviewer: Need to clarify that we don't get CCS from the raw spectral data, but rather after multiple preprocessing steps.

Author: Should be addressed now.

> Consequently, the \gls{ccs} data for each sample forms a substantial high-dimensional matrix with dimensions $50 \times 6144$.

Reviewer: Wouldn't it be $5\times 50 \times 6144$, because we're collecting five sets for each sample?

Author: Fixed

The instrument used to collect this data is a \gls{libs} instrument replicating the \gls{chemcam} instrument on the Curiosity rover.
Both the \gls{chemcam} and laboratory instrument consist of three spectrometers, each producing 2048 channels.
These spectrometers are used to capture the \gls{uv}, \gls{vio}, and \gls{vnir} regions of the spectrum.

> For each sample, five \gls{ccs} datasets are collected by shooting 50 times at the sample from different locations and processing the raw spectral readings\cite{wiensPreflightCalibrationInitial2013}.

Reviewer: ```suggestion
For each sample, five \gls{ccs} datasets are collected by firing 50 laser shots at five different locations on the sample and processing the raw spectral readings \cite{wiensPreflightCalibrationInitial2013}.
```
